lib/whitelistedUrlPrefixes.js

```diff
@@ -28,5 +27,8 @@ module.exports = [
   'https://ads-serve.bravesoftware.com/', // remove this once ads catalog moves to using prod
   'https://pdfjs.robwu.nl/logpdfjs', // allowed because it gets canceled in tracking protection
   'https://publishers-distro.basicattentiontoken.org/',
-  'https://publishers-staging-distro.basicattentiontoken.org/'
+  'https://publishers-staging-distro.basicattentiontoken.org/',
+  // remove this once QA signs off on go-updater changes
```

Comment:
can we wait for QA to sign off on go updater changes before merging this? to avoid having to merge and then remove it.

Reply:
@diracdeltas - that's the plan :). That's why I have the `pr/work-in-progress` label on the issue